examples/misc_controls_orbit.html

```diff
@@ -48,101 +54,215 @@
 
 			if ( ! Detector.webgl ) Detector.addGetWebGLMessage();
 
-			var stats;
+			var radius = 6371;
+			var tilt = 0.41;
+			var rotationSpeed = 0.02;
+
+			var cloudsScale = 1.005;
+			var moonScale = 0.23;
 
+			var MARGIN = 0;
+			var SCREEN_HEIGHT = window.innerHeight - MARGIN * 2;
+			var SCREEN_WIDTH  = window.innerWidth;
+
+			var container, stats;
 			var camera, controls, scene, renderer;
+			var geometry, meshPlanet, meshClouds, meshMoon;
+			var dirLight, pointLight, ambientLight;
+
+			var composer;
+
+			var textureLoader = new THREE.TextureLoader();
+
+			var d, dPlanet, dMoon, dMoonVec = new THREE.Vector3();
+
+			var clock = new THREE.Clock();
 
 			init();
-			render(); // remove when using next line for animation loop (requestAnimationFrame)
-			//animate();
+
+			// render(); // add this when using controls.addEventListener( 'change', render ) for animation loop
+			animate(); // remove when using event listener for animation loop
```

Comment:
How does one use an event listener for an animation loop?

Reply:
That is a good point - I was just adapting the previous comment in the OrbitControls file. I'll change it to be clearer.